components/EventSection.tsx

```diff
@@ -66,52 +72,78 @@ export function EventSection({
           })}
         </div>
       </div>
-      {!dataFuture && <p>No event data</p>}
-      {view === "Upcoming" && (
-        <div className="flex flex-wrap justify-between gap-4">
-          {dataFuture &&
-            dataFuture.slice(0, 4).map((e: DataProps, i: number) => {
-              return (
-                <div
-                  key={i}
-                  className="bg-gray flex gap-6 overflow-hidden column min-w-[30ch]"
-                >
-                  <CalendarEvent
-                    id={e.id}
-                    date_time={e.date_time}
-                    title={e.title}
-                    url={e.url}
-                    date={e.date}
-                    date_utc={e.date_utc}
-                    date2_utc={e.date2_utc}
-                    date_iso={e.date_iso}
-                    description_long={e.description_long}
-                    contact_info={e.contact_info}
-                    description={e.description}
-                  />
-                </div>
-              )
-            })}
-        </div>
-      )}
-      {view === "Weekly" && (
-        <div className="h-0 min-h-[768px] ">
-          <CalendarWeekly
-            today={today}
-            currentDate={currentDate}
-            events={dataPast.concat(dataFuture)}
-          />
+      <div className="flex flex-wrap justify-evenly gap-6">
+        <div className="flex flex-col justify-center items-start min-w-[20%]">
+          <Card className="flex items-start w-full h-full border-none shadow-none">
+            <CardContent className="py-4">
+              <CCVBars />
+              <h3 className="flex items-center font-semibold text-black text-[32px]">
+                {<FaCalendarAlt className="mr-3" />}
+                Events
+              </h3>
+              <p className="italic text-black text-[20px] text-style mt-[12px]">
+                What's next at CCV
+              </p>
+              <Button
+                className="h-[55px] w-[176px] rounded-[50px] font-semibold text-[20px] mt-[34px]"
+                variant="filled_primary"
+              >
+                <a href={events_url} target="_blank" rel="noopener noreferrer">
+                  View All Events
+                </a>
+              </Button>
+            </CardContent>
+          </Card>
         </div>
-      )}
-      {view === "Monthly" && (
-        <div>
-          <CalendarMonth
-            today={today}
-            currentDate={currentDate}
-            events={dataPast.concat(dataFuture)}
-          />
+        <div className="flex-1">
+          {!dataFuture && <p>No event data</p>}
+          {view === "Upcoming" && (
+            <div className="flex flex-wrap justify-between gap-4">
```

Comment:
Adding some padding, I know this is slightly different from the figma, but i think it needs some space

```suggestion
            <div className="flex flex-wrap justify-between gap-4 pt-4">
```